pkg/apis/core/types.go

```diff
@@ -488,6 +496,17 @@ type PersistentVolumeClaimSpec struct {
 	// (Alpha) Using the namespace field of dataSourceRef requires the CrossNamespaceVolumeDataSource feature gate to be enabled.
 	// +optional
 	DataSourceRef *TypedObjectReference
+	// volumeAttributesClassName is the name of the VolumeAttributesClass required by the claim.
+	// If specified, the CSI driver will create or update the volume with the attributes defined
+	// in the corresponding VolumeAttributesClass. Different with storageClassName, it can be
+	// changed after the claim is created but an empty string value is disallowed.
+	// If unspecified and the PVC is unbound, the default VolumeAttributesClass will be retroactively
+	// used if it exists.
```

Comment:
Clarify what "retroactively used" means or maybe select a different term?  It's not clear to me what this is trying to say.